Sources/Datadog/Tracing/Propagation/OpenTracing/TracingHTTPHeaders.swift

```diff
@@ -8,7 +8,7 @@ import Foundation
 
 /// Trace propagation headers as explained in
 /// https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=browserrum#how-are-rum-resources-linked-to-traces
-internal struct TracingHTTPHeaders {
+internal enum TracingHTTPHeaders {
```

Comment:
removes ablity to `.init()` as it acts as a static namespace. Other solution is making the `.init()` private.